go.mod

```diff
@@ -36,7 +36,7 @@ require (
 	github.com/distribution/reference v0.6.0
 	github.com/docker/distribution v2.8.3+incompatible
 	github.com/docker/go-connections v0.6.0
-	github.com/docker/go-events v0.0.0-20250114142523-c867878c5e32
+	github.com/docker/go-events v0.0.0-20250808211157-605354379745
```

Comment:
Any reason not to create git tags?

Reply:
Yeah, I want to start tagging those, but was considering moving it to the moby org; not sure what's best; start tagging first, then move, or the other way around.

cc @austinvazquez I know we were discussing some of that when you were cleaning up that repository last year